apps/perf-test-react-components/.gitignore

```diff
@@ -0,0 +1 @@
+logfiles
```

Comment:
I know this is copy paste from old perf-test app, but it would be nice if we could move gitignores for both in root gitignore. wdyt ?

Reply:
`.log` files are already ignored in the root .gitignore so i'll just delete these.